### ISSUE TITLE
Uncaught TypeError: Cannot read property 'left' of undefined

### ISSUE THREAD

**Issue by wighawag** [mark-hahn/git-diff-popup#1]
had an edited files (many consecutive lines
pressed ctrl-alt-D

**Atom Version**: 0.192.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: [git-diff-popup](https://github.com/mark-hahn/git-diff-popup) package, v0.1.2
### Stack Trace

Uncaught TypeError: Cannot read property 'left' of undefined

```
At C:\Users\d\.atom\packages\git-diff-popup\lib\diff.coffee:123

TypeError: Cannot read property 'left' of undefined
  at Diff.module.exports.Diff.getDiffBox (C:\Users\d\.atom\packages\git-diff-popup\lib\diff.coffee:123:6)
  at PopupView.module.exports.PopupView.setViewPosDim (C:\Users\d\.atom\packages\git-diff-popup\lib\popup-view.coffee:82:52)
  at C:\Users\d\.atom\packages\git-diff-popup\lib\popup-view.coffee:28:26
  at process._tickCallback (node.js:357:13)

```
### Commands

```
     -2:26.3 core:paste (atom-text-editor.editor.mini)
     -2:25.4 core:confirm (atom-text-editor.editor.mini)
  3x -0:30.5 git-diff-popup:toggle (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "one-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
autocomplete-plus, v2.11.0
git-diff-popup, v0.1.2
haxe, v0.6.1
language-haxe, v0.2.1
last-cursor-position, v0.8.0
linter, v0.12.0

# Dev
No dev packages
```

**Comment by mark-hahn** [comment]
Thanks for the report.  I'm sorry this took so long.

Fixed in 0.1.3

**Comment by despairblue** [comment]
happens again, I suppose because of multiple project folders.

**Atom Version**: 1.0.0
**System**: "Arch Linux"
**Thrown From**: [git-diff-popup](https://github.com/mark-hahn/git-diff-popup) package, v0.1.8

### Stack Trace

Uncaught TypeError: Cannot read property 'left' of undefined

```
At /home/despairblue/.atom/packages/git-diff-popup/lib/diff.coffee:127

TypeError: Cannot read property 'left' of undefined
  at Diff.module.exports.Diff.getDiffBox (/home/despairblue/.atom/packages/git-diff-popup/lib/diff.coffee:127:6)
  at PopupView.module.exports.PopupView.setViewPosDim (/home/despairblue/.atom/packages/git-diff-popup/lib/popup-view.coffee:86:52)
  at /home/despairblue/.atom/packages/git-diff-popup/lib/popup-view.coffee:32:26
  at process._tickCallback (node.js:357:13)

```

### Commands

```
     -0:25.4.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:25.4.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:23.9.0 git-diff-popup:toggle (atom-text-editor.editor.is-focused)
     -0:17 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:17 core:cancel (atom-text-editor.editor.is-focused)
     -0:16.1.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:16.1.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:14.1.0 key-binding-resolver:toggle (atom-text-editor.editor.is-focused)
     -0:13.2.0 git-diff-popup:toggle (atom-text-editor.editor.is-focused)
     -0:10.6.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:10.6.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:08.3.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:08.3.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:06.6.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:03.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:03.4.0 git-diff-popup:toggle (atom-text-editor.editor)
```

### Config

``` json
{
  "core": {
    "excludeVcsIgnoredPaths": false,
    "disabledPackages": [
      "atom-color-highlight",
      "atom-pair",
      "atomatigit",
      "autocomplete",
      "autocomplete",
      "es-navigation",
      "filetype-color",
      "grunt-runner",
      "incremental-search",
      "linter-eslint",
      "linter-jshint",
      "linter-scspell",
      "markdown-scroll-sync",
      "metrics",
      "minimap-color-highlight",
      "minimap-find-and-replace",
      "minimap-git-diff",
      "minimap-highlight-selected",
      "minimap",
      "navigate",
      "package-cop",
      "starred-package-installer",
      "symbols-tree-view",
      "tab-smart-sort",
      "Tern",
      "test-status",
      "themed-settings",
      "travis-ci-status",
      "tree-view-breadcrumb",
      "tree-view-open-files",
      "welcome",
      "zentabs",
      "nuclide-click-to-symbol",
      "nuclide-quick-open"
    ],
    "projectHome": "/home/despairblue/vcs/git/",
    "destroyEmptyPanes": false,
    "autoHideMenuBar": true
  }
}
```

### Installed Packages

``` coffee
# User
angularjs, v0.3.0
atom-alignment, v0.12.1
atom-beautify, v0.28.7
atom-fuzzy-grep, v0.4.0
atom-ternjs, v0.6.7
atom-zeal, v0.1.3
autocomplete-emojis, v2.2.2
autocomplete-paths, v1.0.2
block-travel, v1.0.2
bug-report, v0.7.1
color-picker, v2.0.7
compare-files, v0.6.1
editorconfig, v1.0.1
file-icons, v1.5.8
git-diff-popup, v0.1.8
git-history, v3.0.0
git-log, v0.4.1
git-plus, v5.2.2
go-plus, v3.3.12
grammar-token-limit, v0.1.1
highlight-selected, v0.10.1
hyperclick, v0.0.0
indentation-indicator, v0.6.0
javascript-snippets, v1.0.0
jumpy, v2.0.9
language-dockerfile, v0.1.0
language-dot, v1.6.0
language-fish-shell, v1.0.2
language-haskell, v1.1.6
language-latex, v0.6.1
language-nimrod, v0.1.0
language-puppet, v0.12.0
language-tss, v0.4.0
latex, v0.24.2
linter, v1.1.0
linter-coffeelint, v0.3.2
linter-flow, v0.1.6
linter-htmlhint, v0.0.17
linter-js-standard, v1.0.5
linter-js-yaml, v1.1.1
linter-puppet-lint, v0.2.9
linter-shellcheck, v0.0.8
markdown-preview-opener, v0.1.1
nuclide-code-format, v0.0.0
nuclide-diff-view, v0.0.0
nuclide-file-tree, v0.0.0
nuclide-file-watcher, v0.0.0
nuclide-flow, v0.0.0
nuclide-hack, v0.0.0
nuclide-hg-repository, v0.0.0
nuclide-installer, v0.0.0
nuclide-language-hack, v0.0.0
nuclide-remote-projects, v0.0.0
nuclide-type-hint, v0.0.0
package-sync, v0.2.0
pdf-view, v0.22.0
pigments, v0.8.2
project-manager, v1.15.10
react, v0.12.3
regex-railroad-diagram, v0.8.1
swap-panes, v0.2.0
tabs-to-spaces, v0.11.0

# Dev
linter, v1.1.0
```
